src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -248,25 +258,9 @@ public void onTraversalStart() {
             } else {
                 logger.info("Saving temp file from python:" + scoreFile.getAbsolutePath());
             }
-
-            pythonExecutor.sendSynchronousCommand("from keras import backend" + NL);
-            pythonExecutor.sendSynchronousCommand(String.format("backend.set_session(backend.tf.Session(config=backend.tf.ConfigProto(intra_op_parallelism_threads=%d, inter_op_parallelism_threads=%d)))" + NL, intraOpThreads, interOpThreads));
-
             pythonExecutor.sendSynchronousCommand(String.format("tempFile = open('%s', 'w+')" + NL, scoreFile.getAbsolutePath()));
             pythonExecutor.sendSynchronousCommand("import vqsr_cnn" + NL);
-
```

Comment:
Can you move the above `import vqsr_cnn` into the python code like you did with the keras import.

Reply:
not easily, because the `executePythonCommand` calls vqsr_cnn.score_and_write_batch so this python session needs the vqsr_cnn module loaded